tasks/devcontainer.py

```diff
@@ -1,24 +1,27 @@
 """
-vscode namespaced tags
-
-Helpers for getting vscode set up nicely
+Helpers for working with devcontainers
 """
 
 import json
 import os
+import platform as py_platform
+import sys
 from collections import OrderedDict
+from functools import wraps
 from pathlib import Path
 
 from invoke import task
 from invoke.exceptions import Exit
 
 from tasks.build_tags import build_tags, filter_incompatible_tags, get_build_tags, get_default_build_tags
+from tasks.commands.docker import DockerCLI
```

Comment:
```suggestion
from tasks.commands.docker import DockerCLI, AGENT_REPOSITORY_PATH
```
I think you don't need to redefine it below, correct?

Reply:
Yup!